Library/Formula/aeskeyfind.rb

```diff
@@ -0,0 +1,61 @@
+class Aeskeyfind < Formula
+  homepage "https://citp.princeton.edu/research/memory/code/"
+  url "https://citp.princeton.edu/memory-content/src/aeskeyfind-1.0.tar.gz"
+  sha1 "1b7c267be4b5adec54af2adf67a6733e2bb7896a"
+
+  def install
+    system "make"
+    bin.install "aeskeyfind"
```

Comment:
No `make install` provided in the original makefile, so I used `bin.install` in the formula.

Is this correct?